.github/workflows/release.yml

```diff
@@ -87,5 +87,5 @@ jobs:
           yarn publish --registry=https://registry.npmjs.org --no-git-tag-version --new-version ${{ steps.release.outputs.VERSION }}
           cd ../gestalt-datepicker
           yarn publish --registry=https://registry.npmjs.org --no-git-tag-version --new-version ${{ steps.release.outputs.VERSION }}
-
-
+          cd ../eslint-plugin-gestalt
+          yarn publish --registry=https://registry.npmjs.org --no-git-tag-version --new-version ${{ steps.release.outputs.VERSION }}
```

Comment:
Very curious wether this will work since we haven't yet released this package. Would be good to keep an eye out as soon as we merge this PR.

Reply:
Asked about that on slack! I was wondering if I needed to do the first publish manually. It'd be great if this just worked though

Reply:
@jennyscript yes - ideally we would use the @pinterest user on npm. Example repo: https://www.npmjs.com/package/@pinterest/graphql-lint-rules and namespace our packages that way.